server/data/frontendComponentsClient.ts

```diff
@@ -10,6 +10,33 @@ export interface Component {
 
 export type AvailableComponent = 'header' | 'footer'
 
+type CaseLoad = {
+  caseLoadId: string
+  description: string
+  type: string
+  caseloadFunction: string
+  currentlyActive: boolean
+}
+
+type Service = {
+  description: string
+  heading: string
+  href: string
+  id: string
+}
+
+export interface FrontendComponentsMeta {
+  activeCaseLoad: CaseLoad
+  caseLoads: CaseLoad[]
+  services: Service[]
+}
+
+export interface FrontendComponentsResponse {
+  header?: Component
+  footer?: Component
+  meta: FrontendComponentsMeta
+}
+
```

Comment:
I'm confused, is this FrontendComponent stuff needed for the conditional fields? 🤔

Reply:
I added these in along with the `getComponents` as they are used in the service I added which is then used as part of generating the conditional fields.

Reply:
the conditional fields stuff doesn't use `getComponents`

Reply:
I forgot to remove these bits from the client, I can do still if wanted.

Reply:
this is totes fine actually. even though it's not part of the changes in this pr, i noticed that our old way of retrieving the header/footer components is 1) deprecated and 2) this alternative also brings in caseloads (which we will need one way or another)